pkg/ebpf/bytecode/manager.go

```diff
@@ -57,7 +58,7 @@ func NewManager(closedHandler *PerfHandler) *manager.Manager {
 		},
 		Probes: []*manager.Probe{
 			{Section: string(TCPSendMsg)},
-			{Section: string(TCPSendMsgPre410)},
+			{Section: string(TCPSendMsgPre410), MatchFuncName: strings.TrimPrefix(string(TCPSendMsg), "kprobe/")},
```

Comment:
I think we can just set this to `tcp_sendmsg`. `tcp_sendmsg/pre_4_1_0` won't match anything.

Reply:
👍